spec/wibox/test_utils.lua

```diff
@@ -39,20 +39,36 @@ end
 say:set("assertion.widget_fit.positive", "Offering (%s, %s) to widget and expected (%s, %s), but got (%s, %s)")
 assert:register("assertion", "widget_fit", widget_fit, "assertion.widget_fit.positive", "assertion.widget_fit.positive")
 
+local function times(char, len)
+    local chars = {}
+    for _ = 1, len do
+        table.insert(chars, char)
+    end
+    return table.concat(chars, "")
+end
+
+--- Pad the `str` with `len` amount of characters `char`.
+-- When `is_after == true`, the padding will be appended.
+-- This allows exceeding the limit of 99 in `string.format`.
+local function pad_string(str, char, len, is_after)
+    len = len - #str
+    local pad = times(char, len)
+
+    if is_after == true then
```

Comment:
The parameter only makes sense as a boolean. Since we generally don't do `assert`s, I think it makes more sense to consider something non-sensical like `is_after == "foo"` not the same as an explicit `true`.

Reply:
I don't agree with that, but I can understand your POV here. So let's say we agree :+1: